src/applications/vaos/containers/ExpressCareFormPage.jsx

```diff
@@ -1,69 +1,182 @@
+  email: {
+    'ui:title': 'Email address',
+  },
+};
+
+let form;
+
 function ExpressCareFormPage({
-  submitStatus,
-  submitErrorReason,
+  newRequest,
   localWindowString,
-  submitExpressCareRequest,
+  openReasonForRequestPage,
   router,
   routeToPreviousAppointmentPage,
+  schema,
+  submitErrorReason,
+  submitExpressCareRequest,
+  submitStatus,
+  updateFormData,
 }) {
+  useEffect(() => {
+    document.title = `${pageTitle} | Veterans Affairs`;
+    scrollAndFocus();
+    openReasonForRequestPage(pageKey, uiSchema, initialSchema);
+  }, []);
+
+  const { data } = newRequest;
+
   return (
     <div>
-      <h1>Express Care form</h1>
-      <FormButtons
-        backBeforeText=""
-        backButtonText="Back"
-        nextButtonText="Submit Express Care request"
-        pageChangeInProgress={submitStatus === FETCH_STATUS.loading}
-        disabled={submitStatus === FETCH_STATUS.failed}
-        loadingText="Submitting your Express Care request"
-        onBack={() => routeToPreviousAppointmentPage(router, 'form')}
+      <h1>{pageTitle}</h1>
+      <SchemaForm
+        name="Type of appointment"
+        title="Type of appointment"
+        schema={schema || initialSchema}
+        uiSchema={uiSchema}
         onSubmit={() => submitExpressCareRequest(router)}
-      />
-      {submitStatus === FETCH_STATUS.failed && (
-        <>
-          {submitErrorReason === EXPRESS_CARE_ERROR_REASON.error && (
-            <AlertBox
-              status="error"
-              headline="Your request didn’t go through"
-              content={
-                <p>
-                  Something went wrong when we tried to submit your request and
-                  you’ll need to start over. We suggest you wait a day to try
-                  again or you can call your medical center to help with your
-                  request.
-                </p>
-              }
-            />
-          )}
-          {submitErrorReason === EXPRESS_CARE_ERROR_REASON.noActiveFacility && (
-            <AlertBox
-              status="error"
-              headline="Express Care isn’t available right now"
-              content={
-                <p>
-                  Express Care is only available {localWindowString} today. To
-                  use Express Care, check back during the time shown above.
-                </p>
-              }
-            />
-          )}
-        </>
-      )}
+        onChange={newData => updateFormData(pageKey, uiSchema, newData)}
+        data={data}
+      >
+        <FormButtons
+          backButtonText="Back"
+          nextButtonText="Submit Express Care request"
+          pageChangeInProgress={submitStatus === FETCH_STATUS.loading}
+          disabled={submitStatus === FETCH_STATUS.failed}
+          loadingText="Submitting your Express Care request"
+          onBack={() => routeToPreviousAppointmentPage(router, 'form')}
+          onSubmit={() => submitExpressCareRequest(router)}
```

Comment:
If you remove this `onSubmit` prop, the validation should work as you expect. When this prop is left off, FormButtons won't do anything other than trigger a submit event for the form its in, which `SchemaForm` will handle as we expect.

The `onSubmit` prop was added for uses of this button outside of a form, like on the facilities page when only one facility is valid and we show an info box.

Reply:
Thanks.  I knew it had to be something simple like that.